docs/APIRef.DeviceObjectAPI.md

```diff
@@ -247,6 +248,13 @@ if (device.getPlatform() === 'ios') {
   await expect(loopSwitch).toHaveValue('1');
 }
 ```
- 
+
+### `device.pressBack()`
+Simulate press back button (Android only).
```

Comment:
Please add **Android Only** in the title (and link) instead of the description.

Reply:
OK